src/styles.css

```diff
@@ -84,19 +84,27 @@ nav {
 }
 
 main {
-  padding: 16px;
+  /*  padding: 16px; */
+  /*  padding: 116px 90px; */
 }
 
 header {
-  background: var(--color-background);
+  xxxbackground: var(--color-background);
   color: var(--color-text-primary);
-  box-shadow: 0 4px 8px #888888;
+  xxxbox-shadow: 0 4px 8px #888888;
+}
+
+footer {
+  background: #777;
```

Comment:
when using new colors, it helps to define css variables and reuse them. It makes it easy to change the color later:

(at the top of this file)
```css
html {
  --color-text-primary: #2a3443;
  --color-text-secondary: #5e6673;
  --color-background: #db960a;
}
```

It can then be used like:

```css
  color: var(--color-text-primary);
```